docs/content/querying/filters.md

```diff
@@ -150,6 +150,33 @@ Search filters can be used to filter on partial string matches.
 
 The search filter supports the use of extraction functions, see [Filtering with Extraction Functions](#filtering-with-extraction-functions) for details.
 
+#### Search Query Spec
+
+##### Contains
+
+|property|description|required?|
+|--------|-----------|---------|
+|type|This String should always be "contains".|yes|
+|value|A String value to run the search over.|yes|
+|caseSensitive|Whether two string should be compared as case sensitive or not|no (default == false)|
+
+##### Insensitive Contains
+
+|property|description|required?|
+|--------|-----------|---------|
+|type|This String should always be "insensitive_contains".|yes|
+|value|A String value to run the search over.|yes|
+
+Note that an "insensitive_contains" search is equivalent to a "contains" search with "caseSensitive": false (or not
+provided).
```

Comment:
@vogievetsky was insensitive contains deprecated? I just moved this SearchQuerySpec section around since it was separated from the SearchFilter documentation by the InFilter stuff